petastorm/spark/spark_dataset_converter.py

```diff
@@ -234,7 +279,22 @@ def make_spark_converter(
     else:
         compression_codec = "uncompressed"
 
+    # pylint: disable=global-statement
+    global _cache_format
+    # pylint: enable=global-statement
+    if _is_delta_available():
+        _cache_format = "delta"
+    else:
+        _cache_format = "parquet"
+        warnings.warn("The output dataset may read partial data due to S3 "
+                      "eventual consistency delay. We recommend enabling "
```

Comment:
does delta.io solve S3 eventual consistency?

Reply:
No. Opensouce delta do not support S3. from @cloud-fan 's feedback